srv/salt/ceph/openattic/default.sls

```diff
@@ -5,3 +5,13 @@ install openattic:
     - pkgs:
       - openattic
 
+enable openattic-rpcd:
```

Comment:
FYI, openattic-rpcd will be removed in 2.0.19, so this likely means we have to make sure that DeepSea is aware of these changes as well.